api/src/main/java/org/openmrs/module/m2sysbiometrics/service/impl/SearchServiceImpl.java

```diff
@@ -1,35 +1,88 @@
 package org.openmrs.module.m2sysbiometrics.service.impl;
 
-import java.util.List;
-import org.openmrs.module.m2sysbiometrics.bioplugin.BioServerClient;
+import org.openmrs.module.m2sysbiometrics.bioplugin.LocalBioServerClient;
+import org.openmrs.module.m2sysbiometrics.bioplugin.NationalBioServerClient;
+import org.openmrs.module.m2sysbiometrics.exception.M2SysBiometricsException;
 import org.openmrs.module.m2sysbiometrics.model.M2SysCaptureResponse;
 import org.openmrs.module.m2sysbiometrics.model.M2SysResults;
 import org.openmrs.module.m2sysbiometrics.service.SearchService;
 import org.openmrs.module.m2sysbiometrics.xml.XmlResultUtil;
 import org.openmrs.module.registrationcore.api.biometrics.model.BiometricMatch;
 import org.openmrs.module.registrationcore.api.biometrics.model.BiometricSubject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @Component
 public class SearchServiceImpl implements SearchService {
 
-    public List<BiometricMatch> search(M2SysCaptureResponse fingerScan, BioServerClient client) {
-        String response = client.identify(fingerScan.getTemplateData());
+    private static final Logger LOG = LoggerFactory.getLogger(SearchServiceImpl.class);
+
+    @Autowired
+    private LocalBioServerClient localBioServerClient;
+
+    @Autowired
+    private NationalBioServerClient nationalBioServerClient;
+
+    @Override
+    public List<BiometricMatch> searchLocally(M2SysCaptureResponse fingerScan) {
+        List<BiometricMatch> biometricMatches;
```

Comment:
this var doesn't need to be defined, just return on line 40

Reply:
ok I will correct it